src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -9,6 +9,13 @@
 * Update OpenTelemetry SDK version to `1.8.1`.
   ([#1668](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1668))
 
+* Add OTLP protobuf metric exporter for windows environment. Use
+  `"PrivatePreviewOtlpProtobufMetricExporter=true"` in the connection string to opt-in.
```

Comment:
Why is this called `PrivatePreviewOtlpProtobufMetricExporter`? I'm concerned that `OtlpProtobufMetricExporter` is going to create lots of confusion.

Reply:
@reyang - This was a typo. I had changed the name to `PrivatePreviewEnableOtlpProtobufEncoding` in https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1667. Sorry for the confusion. Fixed it here as well.